src/javascript/app/base/footer.js

```diff
@@ -15,8 +15,11 @@ const Footer = (() => {
             if (Client.isLoggedIn()) {
                 showWarning((Client.get('landing_company_shortcode') === 'maltainvest' ||
                     (Client.get('is_virtual') && State.getResponse('landing_company.financial_company.shortcode') === 'maltainvest')));
+                showAgeRestrictionSign(/iom|maltainvest/.test(Client.get('landing_company_shortcode')) ||
+                    (Client.get('is_virtual') && /iom|maltainvest/.test(Client.get('landing_company_shortcode'))));
```

Comment:
I don't see the difference, either way we are checking `/iom|maltainvest/.test(Client.get('landing_company_shortcode'))`, right? why do we need to check `is_virtual` then?

Reply:
my bad, we do not need that